fiftyone/core/dataset.py

```diff
@@ -1778,7 +1778,7 @@ def clear_sample_field(self, field_name):
         The field will remain in the dataset's schema, and all samples will
         have the value ``None`` for the field.
 
-        You can use dot notation (``embedded.field.name``) to clone embedded
+        You can use dot notation (``embedded.field.name``) to clear embedded
         frame fields.
```

Comment:
Good eye but now I think the word "frame" is another copy pasta accident.
```suggestion
        You can use dot notation (``embedded.field.name``) to clear embedded
        sample fields.
```